app/src/main/kotlin/at/bitfire/davdroid/sync/AutomaticSyncManager.kt

```diff
@@ -69,9 +66,9 @@ class AutomaticSyncManager @Inject constructor(
             workerManager.disablePeriodic(account, dataType)
 
         // also enable/disable content-triggered syncs
-        val possibleAuthorities = dataType.possibleAuthorities(context)
+        val possibleAuthorities = dataType.possibleAuthorities()
         val authority: String? = when (dataType) {
-            SyncDataType.CONTACTS -> context.getString(R.string.address_books_authority)
+            SyncDataType.CONTACTS -> ContactsContract.AUTHORITY
```

Comment:
This wouldn't do anything useful, because accounts don't have/sync contacts (this is what local address books are for). So it doesn't do anything to update `ContactsContract.AUTHORITY` sync for `account`.

The logic that integrates the sync framework with local address books is instead done in `LocalAddressBook.updateSyncFrameworkSettings()` (and called by `LocalAddressBook.create()` / `update()`). So for the sync framework integration of contacts, we don't have to do anything here.

Reply:
Ah yes. So an `account` could never be an addressbook account. I should have dug a bit deeper. I will add a comment as well.